maine-thesis.cls

```diff
@@ -79,6 +79,18 @@
 \RequirePackage[none]{hyphenat}[2009/09/02 v2.3c hyphenation utilities]
 \AfterPackage{hyperref}{% Graduate School requirements mandate a slight modification to hyperref's redefinition of the caption command.  This is implemented here.  The command is only executed after hyperref is loaded in the document (if it is loaded).
 \long\def\@caption#1[#2]#3{%
+\hypersetup{ pdfdisplaydoctitle=true}
```

Comment:
Fixed. accidentally used 2 commits, and can't see how to alter that. I can close this pull, and open another if necessary.